common/src/responder_id.rs

```diff
@@ -1,4 +1,5 @@
-// Copyright (c) 2018-2021 The MobileCoin Foundation
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+#![deny(missing_docs)]
```

Comment:
nit: this usually goes in `lib.rs`

Reply:
I tried that, but this is a large package, which I don't want to tackle in this already large PR.